jbpm-console-ng-showcase/src/main/java/org/jbpm/console/ng/client/screens/HomeViewImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
maybe is a good moment to fix the some home page links replacing  "Tasks List" -> "DataSet Tasks"  and "Process Instances" -> "DataSet Process Instances With Variables" and "Process Definition List" ->"Process Definitions".